index.js

```diff
@@ -1,8 +1,42 @@
 /* jshint node: true */
 'use strict';
+var Funnel = require('broccoli-funnel');
+var VersionChecker = require('ember-cli-version-checker');
 
 module.exports = {
   name: 'ember-i18n',
+  isLocalizationFramework: true,
 
-  isLocalizationFramework: true
+  config: function() {
+    var checker = new VersionChecker(this);
+    var dep = checker.for('ember', 'bower');
+    this.hasEmberHelper = dep.satisfies('>= 1.13.0');
```

Comment:
Prereleases do not satisfy this range:

```
# node
> semver = require('semver')
> semver.satisfies('2.9.0-beta.1', '>= 1.13.0')
false
```

You should use `this.hasEmberHelper = !dep.lt('1.13.0');` for this.